examples/pipelines/datacomp/pipeline.py

```diff
@@ -67,15 +66,7 @@
 pipeline.add_op(load_from_hub_op)
 pipeline.add_op(filter_image_resolution_op, dependencies=load_from_hub_op)
 pipeline.add_op(filter_complexity_op, dependencies=filter_image_resolution_op)
-pipeline.add_op(cluster_image_embeddings_op, dependencies=filter_complexity_op)
```

Comment:
The cluster image embeddings component won't be used for now, which is why it is removed here.